src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/IntegrationTestBase.kt

```diff
@@ -1,25 +1,39 @@
 package uk.gov.justice.digital.hmpps.educationandworkplanapi.app
 
+import com.microsoft.applicationinsights.TelemetryClient
+import org.awaitility.Awaitility
 import org.junit.jupiter.api.BeforeEach
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.test.context.SpringBootTest
 import org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT
+import org.springframework.boot.test.mock.mockito.SpyBean
 import org.springframework.test.context.ActiveProfiles
 import org.springframework.test.web.reactive.server.WebTestClient
 import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.repository.ActionPlanRepository
 import java.security.KeyPair
+import java.util.concurrent.TimeUnit.MILLISECONDS
+import java.util.concurrent.TimeUnit.SECONDS
 
 @SpringBootTest(webEnvironment = RANDOM_PORT)
 @ActiveProfiles("integration-test")
 abstract class IntegrationTestBase {
 
+  init {
+    // set awaitility defaults
+    Awaitility.setDefaultPollInterval(500, MILLISECONDS)
+    Awaitility.setDefaultTimeout(3, SECONDS)
+  }
+
   @Suppress("SpringJavaInjectionPointsAutowiringInspection")
   @Autowired
   lateinit var webTestClient: WebTestClient
 
   @Autowired
   lateinit var actionPlanRepository: ActionPlanRepository
 
+  @SpyBean
```

Comment:
I'm not a fan of spy beans, but in this case its the only way I could realistically do it - `TelemetryClient` is not our class, its a bean that is part of MS App Insights, so _what_ it does and _how_ it does it I dont really care. What we want to do is spy on it to make sure the relevant method was called (which you'll see in the integration tests below)